contrib/utilities/indent

```diff
@@ -63,6 +63,18 @@ find tests include source examples \( -name '*.cc' -o -name '*.h' -o -name '*.cu
 # use the same process to set file permissions for all source files
 find tests include source examples \( -name '*.cc' -o -name '*.h' -o -name '*.cu' -o -name '*.cuh' \) -print | xargs -n 50 -P 10 chmod 644
 
+# convert dos formatted files to unix file format by stripping out 
+# carriage returns (15=0x0D):
+dos_to_unix()
+{
+f=$1
+tr -d '\015' <$f >$f.tmp
+diff -q $f $f.tmp >/dev/null || cp $f.tmp $f
+rm $f.tmp
```

Comment:
Just use `|| mv $f.tmp $f` in the first line, and `rm -f` in the second. This avoids copying the file in its entirety. 

Or do
```
  (diff -q $f $f.tmp >/dev/null && rm $f.tmp) || mv $f.tmp $f
```
which I think does exactly what you want (but please double check my boolean logic), although it would require a comment to explain it ;-)